src/core_plugins/tile_map/public/coordinatemap_response_handler.js

```diff
@@ -17,26 +17,12 @@
  * under the License.
  */
 
-import { tabifyAggResponse } from 'ui/agg_response/tabify/tabify';
 import { convertToGeoJson } from 'ui/vis/map/convert_to_geojson';
 
 export function makeGeoJsonResponseHandler() {
 
-  let lastEsResponse;
-  let lastGeoJsonResponse;
+  return function (table) {
 
-  return function (vis, esResponse) {
-    if (lastEsResponse === esResponse) {
```

Comment:
this is handled on visualize_data_loader level, response handler (any) will not be called if input data did not change